xcm/xcm-builder/src/currency_adapter.rs

```diff
@@ -142,6 +142,7 @@ impl<
 	}
 
 	fn deposit_asset(what: &MultiAsset, who: &MultiLocation) -> Result {
+		log::trace!("xcm::currency_adapter deposit_asset {:?} {:?}", what, who);
```

Comment:
still needed?

Reply:
It was added before, i dont see a problem with log messages